app/src/audio-utils/getMicAudioStream.ts

```diff
@@ -0,0 +1,3 @@
+const getMicAudioStream = (): Promise<MediaStream> => navigator.mediaDevices.getUserMedia({ audio: true });
```

Comment:
Will this work for all browsers?  The earlier .js file defined userMedia if it was not implemented in the browser.

Reply:
We have an expectation that users will be using modern browsers so this is not an issue https://caniuse.com/stream

Reply:
Should we still mention that new versions of browsers are recommended in README? Otherwise the users might just end up wondering why is it not working

Reply:
I don't think that's necessary. The most recent release of the major browsers to add support to this was Safari 4 years ago